src/Pkg.jl

```diff
@@ -307,6 +307,7 @@ The result is a `Dict` that maps a package UUID to a `PackageInfo` struct repres
 |:------------------|:-----------------------------------------------------------|
 | name              | The name of the package                                    |
 | version           | The version of the package (this is `Nothing` for stdlibs) |
+| in_project        | The package is in the project file (a direct dependency)   |
```

Comment:
can `in_project` be `is_direct_dep` (or similar) instead? it would be consistent with all the other `is_*` boolean fields. It is also more precise because technically test deps can also be in the project file

Reply:
yeah, makes sene,